docs/source/host-collect-analyze/subnetAvailable.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Subnet Available
+description:  Collect and analyze information about checking for an available (IPv4) subnet.
+---
+
+## Subnet Available Collector
+
+To check if there is an available (IPv4) subnet on a node, you can use the `subnetAvailable` collector. This is useful for Pod/Service CIDR ranges.
+
+This collector searches for overlap with the routing table of the node to help avoid conflicts.
+
+### Parameters
+
+In addition to the [shared collector properties](/collect/collectors/#shared-properties), the `subnetAvailable` collector accepts the following parameters:
+
+#### `CIDRRangeAlloc` (Required)
+The overarching subnet range to search for available CIDR blocks to use. The format must be `"x.x.x.x/y"`, with an IPv4 network and `y` being a CIDR mask between 1 and 32.
+
+#### `desiredCIDR` (Required)
+An integer between 1 and 32.
+
+Searches in `CIDRRangeAlloc` for an IP subnet of this CIDR block size.
+
+### Example Collector Definition
+
+```yaml
+apiVersion: troubleshoot.sh/v1beta2
+kind: HostPreflight
+metadata:
+  name: subnet-available
+spec:
+  collectors:
+    # would output yes/no depending if there is a /22 available in 10.0.0.0/8
+    - subnetAvailable:
```

Comment:
Please update this example in light of [this comment](https://github.com/replicatedhq/troubleshoot/pull/1004#pullrequestreview-1305738327)